packages/block-editor/src/components/block-list/zoom-out-separator.js

```diff
@@ -101,14 +101,15 @@ export function ZoomOutSeparator( {
 		<AnimatePresence>
 			{ isVisible && (
 				<motion.div
-					as="button"
-					layout={ ! isReducedMotion }
```

Comment:
I can’t see why this is currently a button since it doesn’t have interactivity. Besides that if it’s a button I don’t think it should contain a `div`. 

The `layout` prop doesn’t do anything in this context.